src/services/SettingsService.ts

```diff
@@ -1,11 +1,13 @@
+import { Platform } from "obsidian";
+
 export interface ObsidianZoomPluginSettings {
   debug: boolean;
   zoomOnClick: boolean;
 }
 
 const DEFAULT_SETTINGS: ObsidianZoomPluginSettings = {
   debug: false,
-  zoomOnClick: true,
+  zoomOnClick: Platform.isDesktop,
```

Comment:
I'm afraid that's not enough. Obsidian settings are stored in a json file inside the vault and synchronized across devices. if the user enabled zoomOnClick on desktop, it will affect mobile devices. maybe we need to make two settings in the json file, but show a single title on the settings tab?